carma_novatel_driver_wrapper/package.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<!--  
+ Copyright (C) 2021 LEIDOS.
+
+ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ use this file except in compliance with the License. You may obtain a copy of
+ the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ License for the specific language governing permissions and limitations under
+ the License.
+-->
+<package format="3">
+  <name>carma_novatel_driver_wrapper</name>
+  <version>0.1.0</version>
+  <description>Package containing lifecycle controllers for each carma-platform subsystem</description>
+  <author email="Anish.Deva@leidos.com">Anish Deva</author>
+  <maintainer email="carma@dot.gov">CARMA</maintainer>
+  <license>Apache-2.0</license>
+
+  <buildtool_depend condition="$ROS_VERSION == 1">catkin</buildtool_depend>
```

Comment:
You don't need to specify catkin here or in the export section. Since we are only building this in ROS2 (not both 1 and 2), we don't need to make the package xml conditional. Just the one version check in the cmake file is sufficient.